packages/inventory/src/EntityTable.js

```diff
@@ -134,11 +126,22 @@ class EntityTable extends React.Component {
             }];
         }
 
-        return rows
-        .map((oneItem) => ({
-            ...oneItem,
-            cells: this.buildCells(oneItem)
-        }));
+        return flatten(rows
```

Comment:
You could use flat map here. Babel should be able to handle it I think.

Reply:
I would have to import it, because it was removed from Babel 7. And we were told, that some browsers were not fully supporting `flatMap`

Reply:
I meant just `flat` that should be supported by everything now except IE.

Reply:
Oh yeah, that can work. I checked the support matrix and looks like the biggest problem was with FF, they started to support `flat` at version 62, that was released on 2018-05-07, almost 2 years. So we should be fine.